### PR TITLE
feat: Added two classification examples using Vowpal Wabbit

### PR DESCRIPTION
**Pull request by chenhuims**
- One example uses Sentiment140 data for twitter sentiment classification
- The other example applies VW algorithm to the adult census dataset

### CODE REVIEW

File: notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
...GN1k+/VqHE4gzTf5DJgHmlo2gjgJppb2KVdsZmZDVlugJiZdc47SRenR0fEE7VESW9vpnCkZV+nAdMkjSBdfJ8j6YukCdBrgWER8fMWYty7QdpYYHVErMxDsJ7JafX2Jc1zeLwQ81OkuSlTJO0A/JLUY9BXA6TpB1JFxBpJN5EaHmfmn7dHxJOFbA+Texki4pFmj92f3PvxJuDsiLiskP5XjULt5TAdic3MbCjxHBAzs87ZlH9uPtfm5Vrf31/BPPxps0gP7vtdPtaIiNgE/ACYKGnfBuVf2mSMh+ZVrmrl9iL1utyS33cjqRfneEl7FPLtBpxIuvhf10vMa0kX3CP7iWEtsFOT8UIahjVGaTnj/UhzUYpmkBoA5zUqXB/nAGxVn9mn2LrBsTb/rP9cnYrNzGzIcA+ImVl5/T1s7lbSUJ2bJE0GXkwalrSM/udn/FzSo8CvgCdIF9qnAjdGxPqc50zSHfnZ+fiLgJ2B8cBhpHki/VkIzJR0JWlo0Kn55wWFPJ8jTY6+S9LXSBfQHybdyS8uUfugpNtIS/A+RVoy+FjSMsJ9uQ84QtInSd/NwxFxbx/5fwI8DfwraeWoHxR3RsTivNTthXlI2I+ANaQVqCYAV5KWHR6QiFiVnzVydp54v5Q0lGpPtv5duC+nfUHS9BznDzsVm5nZUOIGiJlZeX0OHYqIRZL+EbiYdLG8lHSBuQb4Zj/H+zrwbtLd9VGkYU6X89wD7oiI5UoPAzwPOJ7U4FhBmnNSbBj05WfAHNJzKfYgLaX77ohYVHif+ZLeRJoY/9mcfDcwMSLuLxzrCuAY0vyPkcAjpCVyL+/jcwJ8krTi1MWkSe9Xk+Z3NBQR6yT9hDT86uY87Ks+zyWSFuVjn5uTHwN+SmrAlHUiqYHwsfw5bgGOJtXP5s8VEXfnhsYppGezDCM9P2RpB2MzMxsSFNH00FszM3uekDScdFf+yxHx6cGOx8zMXjg8B8TMzMzMzCrjBoiZmZmZmVXGDRAzsxeuYABL4JqZmbWD54CYmZmZmVll3ANiZmZmZmaVcQPEzMzMzMwq4waImZmZmZlVxg0QMzMzMzOrjBsgZmZmZmZWGTdAzMzMzMysMm6AmJmZmZlZZdwAMTMzMzOzyrgBYmZmZmZmlXEDxMzMzMzMKuMGiJmZmZmZVeb/Aerpa1m22UhkAAAAAElFTkSuQmCC"
```
Comment (drdarshan) [inline notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb]: Please consider removing output cells from all notebooks.

Comment (chenhuims) [reply notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb]: Thanks for your suggestion. I will remove the output cells. For the ROC curve, may I keep it as what we do in this example https://github.com/Azure/mmlspark/blob/master/notebooks/samples/Regression%20-%20Vowpal%20Wabbit%20vs.%20LightGBM%20vs.%20Linear%20Regressor.ipynb ?

Comment (chenhuims) [reply notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb]: Hi Sudarshan, I've removed the output cells per your suggestion. Please let me know if it's ready to be merged to master.

Comment (drdarshan) [reply notebooks/samples/Classification - Twitter Sentiment with Vowpal Wabbit.ipynb]: Thank you!

**Comment by chenhuims** [comment]
my pleasure

**Comment by mhamilton723** [comment]
@chenhuims Looks like the data download and load didnt work. your dispay df shows 0 rows. It might have something to do with the schema you provided. Try with inferSchema set to true instead of making and passing a schema.

**Comment by chenhuims** [comment]
> @chenhuims Looks like the data download and load didnt work. your dispay df shows 0 rows. It might have something to do with the schema you provided. Try with inferSchema set to true instead of making and passing a schema.

Thanks for checking. There was indeed some issue with data downloading. I will try to fix this issue.

**Comment by chenhuims** [comment]
@mhamilton723 I made a fix and the notebook runs without issue on my ADB workspace. However, the E2E test still failed. Could you grant me access to the ADB workspace of the testing pipeline so I can check the detailed logs?

**Pull request merged**